src/vm/arm64/CallDescrWorkerARM64.asm

```diff
@@ -76,6 +77,8 @@ LNoFloatingPoint
         ldr     x9, [x19,#CallDescrData__pTarget]
         blr     x9
 
+        ldr     x19, [fp, 24] ; Fixup corrupted X19 callee preserved register
```

Comment:
After removing the equivalent of this line, there are 43 failures on [Arm64/Unix] tip.

```
./JIT.Directed.Misc.function_pointer.MutualThdRecur-fptr.MutualThdRecur-fptr.sh
./JIT.Directed.tailcall.tailcall.tailcall.sh
./JIT.Methodical.Boxing.misc._dbgconcurgc_il._dbgconcurgc_il.sh
./JIT.Methodical.Boxing.misc._relconcurgc_il._relconcurgc_il.sh
./JIT.Methodical.switch.switch10.switch10.sh
./JIT.Methodical.switch.switch11.switch11.sh
./JIT.Methodical.switch.switch1.switch1.sh
./JIT.Methodical.switch.switch2.switch2.sh
./JIT.Methodical.switch.switch3.switch3.sh
./JIT.Methodical.switch.switch4.switch4.sh
./JIT.Methodical.switch.switch5.switch5.sh
./JIT.Methodical.switch.switch6.switch6.sh
./JIT.Methodical.switch.switch7.switch7.sh
./JIT.Methodical.switch.switch8.switch8.sh
./JIT.Methodical.switch.switch9.switch9.sh
./JIT.Methodical.tailcall._il_dbgrecurse_ep_void._il_dbgrecurse_ep_void.sh
./JIT.Methodical.tailcall._il_dbgtest_2a._il_dbgtest_2a.sh
./JIT.Methodical.tailcall._il_dbgtest_2b._il_dbgtest_2b.sh
./JIT.Methodical.tailcall._il_dbgtest_2c._il_dbgtest_2c.sh
./JIT.Methodical.tailcall._il_dbgtest_implicit._il_dbgtest_implicit.sh
./JIT.Methodical.tailcall._il_dbgtest_mutual_rec._il_dbgtest_mutual_rec.sh
./JIT.Methodical.tailcall._il_dbgtest_switch._il_dbgtest_switch.sh
./JIT.Methodical.tailcall._il_dbgtest_virt._il_dbgtest_virt.sh
./JIT.Methodical.tailcall._il_dbgtest_void._il_dbgtest_void.sh
./JIT.Methodical.tailcall._il_relrecurse_ep_void._il_relrecurse_ep_void.sh
./JIT.Methodical.tailcall._il_reltest_2a._il_reltest_2a.sh
./JIT.Methodical.tailcall._il_reltest_2b._il_reltest_2b.sh
./JIT.Methodical.tailcall._il_reltest_2c._il_reltest_2c.sh
./JIT.Methodical.tailcall._il_reltest_implicit._il_reltest_implicit.sh
./JIT.Methodical.tailcall._il_reltest_mutual_rec._il_reltest_mutual_rec.sh
./JIT.Methodical.tailcall._il_reltest_switch._il_reltest_switch.sh
./JIT.Methodical.tailcall._il_reltest_virt._il_reltest_virt.sh
./JIT.Methodical.tailcall._il_reltest_void._il_reltest_void.sh
./JIT.Regression.CLR-x86-JIT.V1.1-M1-Beta1.b143840.b143840.b143840.sh
./JIT.Regression.CLR-x86-JIT.V1-M12-Beta2.b65423.b65423.b65423.sh
./JIT.Regression.CLR-x86-JIT.V1-QFE.b151440.static-none.static-none.sh
./JIT.Regression.CLR-x86-JIT.V2.0-Beta2.b426654.b426654.b426654.sh
./JIT.Regression.Dev11.dev11_10427.conv_ovf_i4.conv_ovf_i4.sh
./JIT.Regression.VS-ia64-JIT.V1.2-M02.b14364.b14364.b14364.sh
./Loader.classloader.TypeInitialization.CircularCctors.CircularCctorFourThreadsBFI.CircularCctorFourThreadsBFI.sh
./Loader.classloader.TypeInitialization.CircularCctors.CircularCctorThreeThreads01BFI.CircularCctorThreeThreads01BFI.sh
./Loader.classloader.TypeInitialization.CircularCctors.CircularCctorThreeThreads02BFI.CircularCctorThreeThreads02BFI.sh
./Loader.classloader.TypeInitialization.CircularCctors.CircularCctorThreeThreads03BFI.CircularCctorThreeThreads03BFI.sh
```

The failure signatures are not exactly what I remember.  I will attempt to debug a shorter one to see if I can root cause the issue.